RHEL/7/input/oval/enable_x11_forwarding.xml

```diff
@@ -0,0 +1,29 @@
+<def-group>
+  <definition class="compliance" id="enable_x11_forwarding" version="1">
+    <metadata>
+      <title>Enable X11 Forwarding</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+      </affected>
+      <description>Enable X11Forwarding to encrypt X11 remote connections over SSH.</description>
+      <reference source="galford" ref_id="20160410" ref_url="test_attestation" />
+    </metadata>
+    <criteria comment="SSH is not being used or conditions are met"
+    operator="OR">
+      <extend_definition comment="sshd service is disabled"
+      definition_ref="service_sshd_disabled" />
```

Comment:
Since this ``` RHEL/7/input/oval/enable_x11_forwarding.xml ``` OVAL depends on ```service_sshd_disabled``` OVAL check it should be placed into ```RHEL/7/input/oval/oval_5.11``` directory (because ```service_sshd_disabled``` check will be included only in case oscap supports OVAL-5.11). In other words now like is, the ```RHEL-7 make content``` will fail on RHEL-6 with openscap-1.0.x (because ```enable_x11_forwarding``` would be included, but the dependency not).

=>  Move ```RHEL/7/input/oval/enable_x11_forwarding.xml ``` to ```RHEL/7/input/oval/oval_5.11``` directory this to work properly.